Classes/Neos/Neos/Ui/Domain/Model/Changes/MoveAfter.php

```diff
@@ -15,6 +15,19 @@
 
 class MoveAfter extends AbstractMove
 {
+    /**
+     * "Subject" is the to-be-moved node; the "sibling" node is the node after which the "Subject" should be copied.
+     *
+     * @return boolean
+     */
+    public function canApply()
+    {
+        $nodeType = $this->getSubject()->getNodeType();
+
+        return $this->getSiblingNode()->getParent()->isNodeTypeAllowedAsChildNode($nodeType);
```

Comment:
Same here, we could just call `siblingNode` as a `referenceNode`, then it would have consistent naming between move before/after and into. I think that's the terminology we use in the UI itself.